applications/openshift/master/file_owner_openshift_pki_cert_files/rule.yml

```diff
@@ -5,22 +5,25 @@ prodtype: ocp4
 title: 'Verify User Who Owns The OpenShift PKI Certificate Files'
 
 description: |-
-  {{{ describe_file_owner(file="/etc/kubernetes/static-pod-resources/.*/.*/.*/tls\.crt", owner="root") }}}
+  {{{ describe_file_owner(file="/etc/kubernetes/static-pod-resources/*/*/*/tls.crt", owner="root") }}}
 
 rationale: |-
   OpenShift makes use of a number of certificates as part of its operation.
   You should verify the ownership of the directory containing the PKI
   information and all files in that directory to maintain their integrity.
+  The directory and files should be owned by the system administrator.
```

Comment:
This doesn't need to be added to the rationale.

Reply:
Thank you, removed.